procedures/katello/release.md.erb

```diff
@@ -24,9 +24,10 @@
 
 ## Release Packager
 
-- [ ] Update `katello`, `katello-repos` and `rubygem-katello` in [foreman-packaging](https://github.com/theforeman/foreman-packaging) [rpm/<%= short_version %>](https://github.com/theforeman/theforeman-rel-eng/blob/master/bump_rpm_packaging) branch.
+- [ ] Update `katello`, `katello-repos` and `rubygem-katello` in [foreman-packaging](https://github.com/theforeman/foreman-packaging) [rpm/<%= short_version %>](https://github.com/theforeman/theforeman-rel-eng/blob/master/bump_rpm_packaging) branch (replace <%= short_version %> with the matching Foreman version):
+  - [ ] `obal update katello katello-repos rubygem-katello --version <%= version %><% if is_rc %> --prerelease <%= extra %> --release keep<% end %>`
 - [ ] Merge packaging PR once job is green
-- [ ] Release all packages: `obal release katello katello-repos rubygem-katello`
+- [ ] Wait for [Jenkins to build the packages](https://ci.theforeman.org/job/foreman-packaging-rpm-<%= short_version %>-release/) (replace <%= short_version %> with the matching Foreman version)
```

Comment:
For what it's worth, we have https://github.com/theforeman/theforeman-rel-eng/blob/master/ci-jobs which can wait on a Jenkins job to run & complete. It could be enhanced to run this job. In that project we do have the mapping. So it could be a new script that's called as `PROJECT=katello VERSION=<%= short_version %> ./rpm-packaging-release`.

Reply:
I'll see to do that in a follow up